PLATER/requirements.txt

```diff
@@ -1,11 +1,13 @@
 coverage
+bmt==0.6.0
+fastapi==0.63.0
 pyaml==20.3.1
 pytest==5.4.1
 pytest-asyncio==0.14.0
 starlette==0.13.6
 uvicorn==0.11.7
-git+git://github.com/patrickkwang/biolink-model-toolkit@master#egg=bmt
 git+https://github.com/ranking-agent/reasoner.git@main#egg=reasoner
-git+https://github.com/TranslatorSRI/reasoner-pydantic#egg=reasoner-pydantic
-git+https://github.com/patrickkwang/fastapi#egg=fastapi
+git+https://github.com/TranslatorSRI/reasoner-pydantic@v1.1.0#egg=reasoner-pydantic
 httpx
+pytest-httpx
```

Comment:
We developed a similar, but perhaps more flexible, tool for testing Strider: https://github.com/patrickkwang/asgiar

Reply:
sounds good will checkit out and use that

Reply:
Hi @patrickkwang looked into asgiar, but didn't find an easy work around with out refactoring the tests. maybe we can differ this to an issue and track it seperately (?). There are some standing issues that I think we could prioritize before this refactor.